node_filesystem/templates/felix.cfg.template

```diff
@@ -1,5 +1,6 @@
 [global]
 MetadataAddr  = None
+IpInIpEnabled = True
```

Comment:
Does this change mean that for calico-docker, we will always get an tunl0 device created? I don't think that's what we want...